test/check-application

```diff
@@ -75,8 +75,7 @@ class TestApplication(testlib.MachineCase):
         self.has_selinux = self.machine.image not in ["debian-testing"]
 
     def cgroupsV2(self):
-        # We will introduce images that are still cgroupsV1 later on
-        return True
+        return self.machine.image not in ["debian-testing"]
```

Comment:
Hm, so "hybrid" is not "v2" enough? Both APIs should be available with that. Or is that just due to the outdated conmon?

Reply:
Does not seems so. At least not for providing stats for rootless containers.